charts/evm-rollup/files/genesis/geth-genesis.json

```diff
@@ -42,6 +42,7 @@
         "astriaEIP1559Params": {{ toPrettyJson .Values.config.rollup.genesis.eip1559Params | indent 8 | trim }}
         {{- if not .Values.global.dev }}
         {{- else }}
+        "astriaSequencerHrpPrefix": "{{ .Values.config.sequencer.addressPrefixes.base }}",
```

Comment:
Should allow astriaBridgeSenderAddress configurations?

Reply:
Updated